pkg/integrationteststatus/integrationteststatus_enumer.go

```diff
@@ -19,6 +19,19 @@ func (i IntegrationTestStatus) String() string {
 	return _IntegrationTestStatusName[_IntegrationTestStatusIndex[i]:_IntegrationTestStatusIndex[i+1]]
 }
 
+func (i IntegrationTestStatus) IsFinal() bool{
```

Comment:
Can this function go here since it's an [autogenerated file](https://github.com/redhat-appstudio/integration-service/blob/1d350630b09babccb009f7503592aa906e5e33ae/pkg/integrationteststatus/integrationteststatus_enumer.go#L1)?

Reply:
No, it should be in `pkg/integrationteststatus/integration_test_status.go`